Info-Orbs/src/main.cpp

```diff
@@ -9,6 +9,11 @@
 #include <globalTime.h>
 #include <config.h>
 #include <widgets/stockWidget.h>
+// #include <AsyncTCP.h>
+// #include <ESPAsyncWebServer.h>
+// #include <WebServer.h>
+// #define ESP32
+// #include <NetWizard.h>
```

Comment:
Testing leftovers?

Reply:
Ah, yes, I tried a few other methods for doing WiFi setup before settling on WiFiManager. Sorry for the cruft.